.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/ucyo/Environments/juplab/bin/python"
+}
```

Comment:
I think .vscode should be on the gitignore; and then not checking this into git :hugs:

Reply:
ALSO Y U NO NEOVIM :scream_cat:

Reply:
Damn! Like a noob... :facepalm: Not that there is anything wrong with that... :)

Reply:
> ALSO Y U NO NEOVIM scream_cat

Be happy its not emacs :smiling_imp: